app/domain/errors.rb

```diff
@@ -542,6 +542,11 @@ module AuthnJwt
         msg: "Failed to parse mapping claims, {0-purpose} value '{1-claim-value}' appears twice or more",
         code: "CONJ00113E"
       )
+
+      RoleWithStandardOrMappedClaimError = ::Util::TrackableErrorClass.new(
+        msg: "Role can't have standard claim or a mapped claim. Error : '{0-error}'",
```

Comment:
```suggestion
        msg: "Role can't have registered claim or a mapped claim. Error : '{0-error}'",
```
https://datatracker.ietf.org/doc/html/rfc7519#section-4.1